index.js

```diff
@@ -24,24 +23,32 @@ function wrap(fn) {
   });
   return copyFnProps(fn, newFn);
 }
-
 function patchRouterParam() {
-  const originalParam = Router.prototype.constructor.param;
-  Router.prototype.constructor.param = function param(name, fn) {
+  const originalParam = express.Router.prototype.constructor.param;
+  express.Router.prototype.constructor.param = function param(name, fn) {
     fn = wrap(fn);
     return originalParam.call(this, name, fn);
   };
 }
 
-Object.defineProperty(Layer.prototype, 'handle', {
-  enumerable: true,
-  get() {
-    return this.__handle;
-  },
-  set(fn) {
-    fn = wrap(fn);
-    this.__handle = fn;
-  },
-});
-
 patchRouterParam();
+
+if (expressVersion[0] > 4) {
+  // eslint-disable-next-line no-console
+  console.warn(`deprecated: Package express-async-errors works with version 4.x.x of Express, you are using ${expressVersion} which supports async route handlers natively`);
```

Comment:
This dep notice should be updated with guidance for what users of this package should do when upgrading to v5. Currently, from a user's perspective, nothing will change. They get the same features that they enjoyed while using this package under Express v4, this is done by only applying the promise support for `Router.param` which this package introduces, and skipping over adding promise support for middleware/route handlers, which Express v5 provides out of the box.